rotcon/convolve_pre.py

```diff
@@ -120,10 +121,11 @@ def __init__(self, *args, **kwargs):
                         S1.write(rotamersSite1.atoms)
                         for nh in proteinHN:
                             atom = proteinHN.select_atoms('resid {}'.format(nh.resnum))
-                            (a, b, distance) = \
+                            (a, b, distance_nitrogen) = \
                                 MDAnalysis.analysis.distances.dist(rotamer1nitrogen, atom)
-                            distances.append([nh.resnum, distance[0]])
-
+                            (a, b, distance_oxygen) = \
```

Comment:
This would be a point where you could calculate either the old way:
```python
if use_NO_electron:
   a, b, distance_oxygen = MDAnalysis.analysis.distances.dist(rotamer1oxygen, atom)
   distance = np.mean([distance_nitrogen[0], distance_oxygen[0]])
else:
   distance = distance_nitrogen[0]
distances.append([nh.resnum, distance])
```
     distance = distance_nitrogen